user_interface/lambda_function.py

```diff
@@ -193,21 +207,24 @@ def set_reminder_frequency(user_reminder: UserReminder, frequency: str) -> None:
     :param user_reminder: UserReminder object for current user
     :return: None
     """
-    chat_id = user_reminder.user_id
-    if frequency.strip().isdigit():
+    user_id = user_reminder.user_id
+    if user_reminder.temp_reminder.name == DEFAULT_NAME:
+        bot.send_message(
+            user_id, 
+            text=f'Please, start from command /create ReminderName')
+    elif frequency.strip().isdigit():
```

Comment:
Why not take in an int as the argument to this function (and below) instead of a string?

Reply:
This functions in general should handle 2 cases with the same message: when user input not an number or incorrect number. So I decided to handle both cases in one place.